examples/v1alpha1/README.md

```diff
@@ -369,7 +369,7 @@ batch2 accuracy=0.8
 Validation-accuracy=0.75
 ```
 The metrics collector will collect all logs of metrics.
-The manifest of metrics collector is also generated from template and defined [here](/manifests/studyjobcontroller/metricsControllerConfigMap.yaml).
+The manifest of metrics collector is also generated from template and defined [here](./manifests/v1alpha1/studyjobcontroller/metricsControllerConfigMap.yaml).
```

Comment:
As well, I found this line with wrong link: 
https://github.com/kubeflow/katib/pull/452/files#diff-be7f7f1b07a698ff4c53ad0e30462b67R202